main/src/library/flix/channel/Channel.flix

```diff
@@ -0,0 +1,18 @@
+namespace Channel {
+
+    ///
+    /// Parallel implementation of `List.map`.
+    ///
+    pub def parMap[a, b](f: a -> b, xs: List[a]): List[b] =
```

Comment:
Whoops. This is not correct. Can you spot the error?

Reply:
Will the elements potentially get shuffled around?

Reply:
Yes, it breaks the order.